src/issues/util.ts

```diff
@@ -364,6 +364,13 @@ export async function createGithubPermalink(gitAPI: GitApiImpl, positionInfo?: N
 	return { permalink: `https://github.com/${new Protocol(upstream.fetchUrl).nameWithOwner}/blob/${log[0].hash}${pathSegment}#L${range.start.line + 1}-L${range.end.line + 1}`, error: undefined };
 }
 
+export function sanitizeIssueTitle(title: string): string {
+	return title
+		.replace(/\W+/g, '-')

```

Comment:
I think this is a little bit too aggressive - if the title is entirely non English characters, it would be completely blanked out. How about keeping the old regex but adding \', \", and \. to it?

Reply:
I think you are right! I restored the original regex and added a few more characters to it.

```
%
.
,
:
;
'
"
~
#
```